ios/ci_scripts/ci_pre_xcodebuild.sh

```
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+# Make script executable
+chmod +x "${BASH_SOURCE[0]}"
```

Shellcheck

[warning] In POSIX sh, BASH_SOURCE is undefined.

(SC3028)